01-messaging.md

```diff
@@ -156,6 +156,7 @@ A sending node:
   - when sending `error`:
     - MUST fail the channel referred to by the error message.
   - SHOULD send `error` for protocol violations or internal errors that make channels unusable or that make further communication unusable.
+  - SHOULD send `error` in reply to messages of type `32`-`255` related to unknown channels.
```

Comment:
SHOULD send _with_ the unknown channel ID

Reply:
fixed!